src/testcafe/testcafe.builder.ts

```diff
@@ -99,7 +99,7 @@ async function execute (
 
         const failedCount = await runTests(testCafe, options);
 
-        if (failedCount > 0)
+        if (failedCount as number > 0)
```

Comment:
Have you tried to change the returning value for the `runTest` function, for example, `Promise<number>`? It's better not to use the `as` construction